3p/integration.js

```diff
@@ -138,9 +138,16 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
-const data = parseFragment(location.hash);
-window.context = data._context;
-
+const iframeName = window.name;
+let data;
+try {
+  data = JSON.parse(iframeName).attributes;
+  window.context = data._context;
```

Comment:
This is now at `data.attributes.context`, per https://github.com/ampproject/amphtml/pull/6498/files#diff-34e84f9cf5e8134d86bf1fc893ab293aR128.